test/repo-init-integration/expected/core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -0,0 +1,2 @@
+clusters:
+  default: api.ci
```

Comment:
We're going to need something *way* more complicated to test this :)

Reply:
(Tested with the config defined https://github.com/openshift/release/pull/8209/files)

Will add unit tests when the core code goes review.

Reply:
Unit and integration tests should exist before review, not after -- they are just as important as code.

Reply:
I am not saying it is not as important.
Just feel that it is more efficient after the core code is more stable.
I meant to do it with the current PR, not following up.

Reply:
Done.